docs/modules/ic-identity-guide/pages/auth-how-to.adoc

```diff
@@ -1,106 +1,106 @@
-= How to use the Internet Identity 
+= How to use Internet Identity
 :keywords: Internet Computer,blockchain,protocol,replica,subnet,data center,canister,developer
 :proglang: Motoko
 :platform: Internet Computer platform
 :IC: Internet Computer
 :company-id: DFINITY
 :sdk-short-name: DFINITY Canister SDK
 
-The Internet Identity enables you to authenticate securely and anonymously when you access applications that use it as an authentication method. A different identity is created for each application you log in to, and you will be able to use all of your registered devices or authentication methods to log in to the same account. This is all accomplished using encrypted smart contracts that run on the {IC} blockchain.
+_Internet Identity_ is a smart contract implemented on the {IC} blockchain. It enables you to authenticate securely and anonymously when you access applications that use it for authentication. An account is associated with an _Identity Anchor_, and a user is free to set up multiple of these. For each application you authenticate to using an Identity Anchor, a different _pseudonym_ specific to that application is created. This pseudonym cannot be linked to the Identity Anchor. You can add multiple _authentication methods_, such as security keys or facial recognition on computers or phones to an Identity Anchor, and you will be able to use all of the added authentication methods to authenticate using that Identity Anchor.
```

Comment:
Thanks for the review @Dfinity-Bjoern. 

> I think I would drop the "... that use it for authentication." The reason is that an application does not necessarily have to know about Internet Identity at all – it could be legitimately be implemented in a browser extension or so.

Agreed, removed this part. 

>  I think we may also not want to use the term account that prominently, but rather start with the Identity Anchor as the core concept.

Good point. I switched to: 
"To authenticate, a user must specify an _Identity Anchor_. A user is free to create multiple of these." 

I also thought about this alternative:
"An identity is associated with an _Identity Anchor_, and a user is free to set up multiple of these."
But I decided against it because here "identity' is just some undefined, intuitive concept.